blog-posts/cdk-scheduler-presentation/cdk-scheduler-presentation.md

```diff
@@ -0,0 +1,47 @@
+---
+published: false
+title: 'CDK Scheduler'
+cover_image:
+description: 'Description of the article'
+tags: serverless, scheduling, cdk, cdk-construct
+series:
+canonical_url:
+---
+
+# CDK Scheduler -- Insert catchy title here
+
+When Guillaume worked on a serverless CMS, he had to find a way to schedule an event (publishing a new article) at a precise time with serverless tools. AWS doesn't offer a built-in service to do so. There are straight-forward options but none are cost-effective and precise. We published a CDK construct that is both for you to add to your stack.
```

Comment:
Je pense qu'il y a une coquille dans cette phrase : We published a CDK construct that is both for you to add to your stackv

Reply:
Peut-être rajouter un TLDR avec un lien vers votre repo

Reply:
J'ai ajouté